clusters/app.ci/prow-monitoring/mixins/_prometheus/dptp_alerts.libsonnet

```diff
@@ -160,6 +160,33 @@
           }
         ],
       },
+      {
+        name: 'openshift-priv-image-building-jobs-failing',
+        rules: [
+          {
+            alert: 'openshift-priv-image-building-jobs-failing',
+            expr: |||
+             (
+               sum(
+                 rate(prowjob_state_transitions{job="prow-controller-manager",job_name=~".*-images",org="openshift-priv",state="failure"}[6h])
```

Comment:
I think `6h` is not good enough. I suggest `12h` because the jobs are running based on what is merged on the public repository and when the sync job will run to update all of them.